test/registry/AddressRegistry.t.sol

```diff
@@ -168,6 +168,15 @@ contract MainnetAddressRegistryTest is Test {
             assertNotEq(addresses.getAddress("ProxyAdminOwner", chainId), address(0), "ProxyAdminOwner not loaded");
             assertNotEq(addresses.getAddress("SystemConfigOwner", chainId), address(0), "SystemConfigOwner not loaded");
             assertNotEq(addresses.getAddress("UnsafeBlockSigner", chainId), address(0), "UnsafeBlockSigner not loaded");
+            assertNotEq(
```

Comment:
We should use `assertEq` throughout here and hardcode the expected result. Otherwise the rest of this PR LGTM

Reply:
Looks like the redundant `assertNotEq` are still present? I'm ok with creating a tracking issue to clean up these tests in a follow up PR, that handles this + https://github.com/ethereum-optimism/superchain-ops/pull/623/files#r1961804263